pennylane/plugins/default_qubit.py

```diff
@@ -334,12 +334,5 @@ def analytic_probability(self, wires=None):
         if self._state is None:
             return None
 
```

Comment:
This was a bug oncovered by the tests: this function should rather just pass on the wires...